### ISSUE TITLE
Header without value causes NoSuchElementException

### ISSUE THREAD

**Issue by mabn** [opentracing-contrib/java-web-servlet-filter#21]
This happened:

```
java.util.NoSuchElementException
1 at java.util.ArrayList$Itr.next (ArrayList.java:854)
2 at io.opentracing.contrib.web.servlet.filter.HttpServletRequestExtractAdapter$MultivaluedMapFlatIterator.next (HttpServletRequestExtractAdapter.java:84)
3 at io.opentracing.contrib.web.servlet.filter.HttpServletRequestExtractAdapter$MultivaluedMapFlatIterator.next (HttpServletRequestExtractAdapter.java:58)
```

The request came with header without values:
```
Accept: application/json
Broken-Header: 
Content-Type: application/json
```

There's a [test](https://github.com/opentracing-contrib/java-web-servlet-filter/blob/master/opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/MultivaluedMapFlatIteratorTest.java#L36) for this case with comment that "this should not really happen" but it happens :)

I was running 0.0.3, but 0.0.7 seems to have the same logic.

**Comment by jakerobb** [comment]
Seems like this can be closed as a duplicate of #22.

**Comment by pavolloffay** [comment]
@jakerobb yes, thanks, closing
